flink-end-to-end-tests/flink-batch-sql-test/src/main/java/org/apache/flink/sql/tests/BatchSQLTestProgram.java

```diff
@@ -48,7 +48,7 @@
  *
  * <p>Parameters:
  * -outputPath output file path for CsvTableSink;
- * -sqlStatement SQL statement that will be executed as sqlUpdate
+ * -sqlStatement SQL statement that will be executed as executeSql
```

Comment:
Yes, I had replace all except `sqlUpdate` method in sql-client and `sqlUpdate` method in old planner.

Regarding (1), we can't replace `sqlUpdate` with `executeSql ` now, because we need to do more refactor work about this. (such as: submit job logic)

Regarding (2), them are in old planner